deno.json

```diff
@@ -7,6 +7,8 @@
     "hono": "jsr:@hono/hono@^4.7.2",
     "hono/logger": "jsr:@hono/hono@^4.7.2/logger",
     "hono/bearer-auth": "jsr:@hono/hono@^4.7.2/bearer-auth",
+    "prom-client": "npm:prom-client@^15.1.3",
+    "metrics": "./src/lib/helpers/metrics.ts",
```

Comment:
you don't need to define metrics in the deno.json. We define some .ts file like this because we allow those files to be overridden using imports, but here you are not allowing that.

https://github.com/iv-org/invidious-companion/blob/8b967d5d39fe5782e63f395091070b6c50606fab/src/lib/helpers/youtubePlayerHandling.ts#L6-L17